frontend/src/organisms/BurgerUser.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { route } from 'src/Routes';
+import { NavLink, Button } from 'src/atoms/';
+import { useAuth } from 'src/utils/auth';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
+export function BurgerUser() {
```

Comment:
tohle není úplně š%tastně pojmenované, když s ena to kouknu tak je to nějaké UserSettingsMenu

Reply:
neo ještě lépe, `UserMenu`?